src/commands/profiles/create.js

```diff
@@ -241,14 +231,11 @@ class ProfilesCreate extends BaseCommand {
       throw new TwilioCliError('Could not create an API Key.');
     }
 
-    this.userConfig.addProfile(this.profileId, this.accountSid, this.region);
-    await this.secureStorage.saveCredentials(this.profileId, apiKey.sid, apiKey.secret);
```

Comment:
[Self] Still keeping `saveCredentials` method in secureStorage intact, and will be removed later on with complete migration.